RockWeb/Licenses/ClearScript.main.license

```diff
@@ -0,0 +1,34 @@
+From https://clearscript.codeplex.com/license
+

```

Comment:
We usually add all our third-party licences to the RockWeb/Licence.aspx page rather than creating files. Is there a benefit to having it as a file in licenses folder?